src/main/resources/db/migration/V1_50_1__contract_type_data.sql

```diff
@@ -0,0 +1,19 @@
+insert into contract_type(id, name, code)
+Values  ('72e60faf-b8e5-4699-9d7c-aef631cca71b', 'Accomodation', 'ACC'),
```

Comment:
In lieu of procurement-owned codes, should we use the same as the ones in Delius? I.e. `ACC` &rarr; `CRS01` etc?